### PR TITLE
Updated footer

### PR DESCRIPTION
**Pull request by rmykolas**
Footer update with the newsletter signup button

### CODE REVIEW

**Comment by dajohi** [comment]
@rmykolas Is it intentional to remove the other links from the footer?

File: src/layouts/partials/footer.html [hunk 1]
```diff
@@ -2,43 +2,27 @@
...
+<div class="footer-bottom">
+    <div class="width1100 footer__container">
+        <p>© 2019 Decred Developers, All Rights Reserved.</p>
+        <div>
+            {{ with $current_release }}
+                <p>Stable Release <span>{{ .current_version }}</span></p>
+            {{ end }}
+            <p>Downloads <span>437k</span></p>
+        </div>
+            <a href="http://explorer.dcrdata.org/">Block Explorer ›</a>
```
Comment (jholdstock) [inline src/layouts/partials/footer.html]: https on the link please

File: src/layouts/partials/footer.html [hunk 2]
```diff
@@ -2,43 +2,27 @@
...
-                {{ range $.Site.Params.social_media_links }}
-                    <a target="_blank" rel="noopener noreferrer" href="{{ .url }}" data-label="{{ .label }}" class="icon w-inline-block {{ .id }}"></a>
-                {{ end }}
-                <div class="backgroundlightblue borderradius4 colorblue dropshadow font12 footerblockindicator opacity000 transition"></div>
-            </div>
+    <div class="footer__container width1100">
+        <ul class="footer__menu">
+            <li><a href="{{ "brand/" | relLangURL }}">{{ T "nav_brand_resources" }}</a></li>
+            <li><a href="{{ "mailto:press@decred.org" }}">Press Contacts</a></li>
+            <li><a href="{{ "https://docs.decred.org/about/license/" | relLangURL }}">Licensing Information</a></li>
```
Comment (jholdstock) [inline src/layouts/partials/footer.html]: Two points here:

1. External URLs do not need to be run through `relLangURL`
1. This link actually goes to the licensing information specifically for dcrdocs. This is not a project-wide license.

**Comment by linnutee** [comment]
Yep that's deliberate. I've mentioned about this in matrix also, those links will be housed in the updated navigation (which is otherwise completed and only needs a PR). The PR will be made as soon as a couple new subpages have content finalized – those can hopefully be implemented quite quickly. Right now the reason to push the footer live asap is providing access to the newsletter signup. 

https://github.com/decred/dcrdesign/issues/128
https://github.com/decred/dcrdesign/issues/129

File: src/layouts/partials/footer.html (hunk 1 above)
Comment (dajohi) [reply src/layouts/partials/footer.html]: dcrdata.decred.org
also, noopenner noreferrer is needed on all these links

File: src/layouts/partials/footer.html (hunk 2 above)
Comment (dajohi) [reply src/layouts/partials/footer.html]: rel="noopener noreferrer" on all links.

**Comment by rmykolas** [comment]
The requested changes have been made @dajohi

**Comment by rmykolas** [comment]
@dajohi just wanted to push this up in your inbox - let me know if there are any other changes needed here and whether this can be merged!

**Pull request closed without merging**